lib/mock_redis/zset_methods.rb

```diff
@@ -118,7 +118,11 @@ def zinterstore(destination, keys, options = {})
 
     def zrange(key, start, stop, options = {})
       with_zset_at(key) do |z|
-        to_response(z.sorted[start.to_i..stop.to_i] || [], options)
+        start = start.to_i
+        stop = stop.to_i
+        beg = start < 0 ? [start, -z.sorted.size].max : start
```

Comment:
I think you could remove this ternary and just do `[start, -z.sorted.size].max` directly.